### PR TITLE
Switch

### PR DESCRIPTION
**Pull request by newshahn**
Hi,
Switch chapter was translated :)

### CODE REVIEW

File: 1-js/02-first-steps/14-switch/1-rewrite-switch-if-else/task.md [hunk 1]
```diff
@@ -1,26 +1,25 @@
-importance: 5
+اهمیت: 5
```
Comment (mehradsadeghi) [inline 1-js/02-first-steps/14-switch/1-rewrite-switch-if-else/task.md:1]: As you can see in this [link](https://github.com/javascript-tutorial/fa.javascript.info#metadata), You should not translate metadata like `importance`.

File: 1-js/02-first-steps/14-switch/2-rewrite-if-switch/task.md [hunk 2]
```diff
@@ -1,23 +1,22 @@
-importance: 4
+اهمیت: 4
```
Comment (mehradsadeghi) [inline 1-js/02-first-steps/14-switch/2-rewrite-if-switch/task.md:1]: Same here.

**Comment by newshahn** [comment]
As you can probably tell, I'm pretty new to programming and Github and that's partly why I made these mistakes.
I will fix them and resend my PR.
Thank you

**Comment by mehradsadeghi** [comment]
> As you can probably tell, I'm pretty new to programming and Github and that's partly why I made these mistakes.
> I will fix them and resend my PR.
> Thank you

Don't worry about it. You're doing fine.

**Comment by newshahn** [comment]
Dear Mehrad,
I fixed the issues you mentioned in the past PR.
An extension was changing my double quotes to single quotes.
Hopefully, it's all okay now.

**Pull request merged**